third_party/gtp_ovs/ovs-gtp-patches/2.15/0024-QFI-support-in-OVS.patch

```diff
@@ -255,6 +256,16 @@ index 57d7a128e..04e26ef55 100644
  };
  
  enum {
+diff --git a/debian/changelog b/debian/changelog
+index 78fe2ab32..431f401ec 100644
+--- a/debian/changelog
++++ b/debian/changelog
+@@ -1,4 +1,4 @@
+-openvswitch (2.15.4-8) unstable; urgency=low
++openvswitch (2.15.4-9) unstable; urgency=low
```

Comment:
I think it would be good to add some more data to the version to make its origin clear:
```suggestion
+openvswitch (2.15.4-9-magma) unstable; urgency=low
```

Reply:
@maxhbr , changes done and generated below packages:

```
libopenvswitch_2.15.4-9-magma_amd64.deb
openvswitch-dbg_2.15.4-9-magma_amd64.deb
openvswitch-testcontroller_2.15.4-9-magma_amd64.deb
libopenvswitch-dev_2.15.4-9-magma_amd64.deb
openvswitch-ipsec_2.15.4-9-magma_amd64.deb
openvswitch-vtep_2.15.4-9-magma_amd64.deb
openvswitch-common_2.15.4-9-magma_amd64.deb
openvswitch-pki_2.15.4-9-magma_all.deb
python3-openvswitch_2.15.4-9-magma_all.deb
openvswitch-datapath-dkms_2.15.4-9-magma_all.deb 
openvswitch-switch_2.15.4-9-magma_amd64.deb
openvswitch-datapath-source_2.15.4-9-magma_all.deb
openvswitch-test_2.15.4-9-magma_all.deb
```